src/oss-find-source/FindSourceTool.cs

```diff
@@ -49,28 +77,28 @@ static void Main(string[] args)
                     {
                         var purl = new PackageURL(target);
                         var results = findSourceTool.FindSource(purl).Result.ToList();
-                        results.Sort((a, b) => (a.Value.CompareTo(b.Value)));
+                        results.Sort((a, b) => a.Value.CompareTo(b.Value));
                         results.Reverse();
```

Comment:
Couldn't you just reverse the sort above instead of sorting then reversing?

Reply:
makes sense...changed it to 
`                        results.Sort((b, a) => a.Value.CompareTo(b.Value));`